getting_started.md

```diff
@@ -52,74 +52,84 @@ using Flux
+A loss function evaluates a machine learning model's performance. In other words, it measures how far the model is from its target prediction. Flux lets you to define your own custom loss function, or you can use one of the [Loss Functions](https://fluxml.ai/Flux.jl/stable/training/training/#Loss-Functions-1) that Flux provides. 
 
-For this example, define a custom loss function:
+For this example, we'll define a loss function that measures the squared distance from the predicted output to the actual output:
 
 ```julia
 function loss(x, y)
-  ŷ = model(x)
-  sum((y .- ŷ).^2)
+  ŷ = training_model(x)
+  sum((y - ŷ).^2)
 end
 ```
-<br>
-
-This function computes the model's prediction for the input `x` and returns the loss for the output `ŷ`.
 
 <br>
 
 ### Step 5: Set an optimiser
 
 You train a machine learning model by running an optimization algorithm (optimiser) that finds the best parameters (`W` and `b`). The best parameters for a model are the ones that achieve the best score of the `loss` function. Flux provides [Optimisers](https://fluxml.ai/Flux.jl/stable/training/optimisers/) that you can use to train a model. 
 
-Set a classic gradient descent optimiser with learning rate η = 0.1:
+For this tutorial, we'll use a classic gradient descent optimiser with learning rate η = 0.01:
 
 ```julia
-opt = Descent(0.1)
+opt = Descent(0.01)
 ```
 <br>
 
 ### Step 6: Train your model
 
-Training a model is the process of computing the gradients with respect to the parameters for each data point in the data. At every step, the optimiser updates all of the parameters until it finds a good value for them. In fact, you can write this process as a *for loop*. Notice that because we are iterating over our training set, we place `x` and `y` into a vector. If we had more points in the training set, they would also be placed in `data`. Also, you need to set `ps = params([W, b])` to indicate that you want the derivatives of `W` and `b`.
+Training a model is the process of computing the gradients with respect to the parameters for each input in the data. At every step, the optimiser updates all of the parameters until it finds a good value for them. This process can be written as a loop: we iterate over the examples in `x_train` and `y_train` and update the model for each example.
 
-You can execute the training process of your model as follows:
+To indicate that we want all derivatives of `W_learned` and `b_learned`, we write `ps = params([W, b])`. This is a convenience function that Flux provides so that we don't have to explicitly list every gradient we want. Check out the section on [Taking Gradients](https://fluxml.ai/Flux.jl/stable/models/basics/#Taking-Gradients) if you want to learn more about how this works.
```

Comment:
```suggestion
To indicate that we want all derivatives of `W_learned` and `b_learned`, we write `ps = params(W, b)`. This is a convenience function that Flux provides so that we don't have to explicitly list every gradient we want. Check out the section on [Taking Gradients](https://fluxml.ai/Flux.jl/stable/models/basics/#Taking-Gradients) if you want to learn more about how this works.
```

Reply:
D'oh. Thanks for the catch.